mandatory/4-tax.js

```diff
@@ -17,8 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
-
+function addTaxAndFormatCurrency(price) {
+  return `£${calculateSalesTax(price).toFixed(2)}`;
```

Comment:
Possibly think about adding another variable in here to make the final return value a bit more readable. 
Which part of the expression `calculateSalesTax(price).toFixed(2)` could you lift out of the template literal and give a meaningful name to ?